### PR TITLE
Core: Remove unused babel dependencies

### PR DESCRIPTION
**Pull request by ndelangen**
## What I did

I think it's obvious, I removed dependencies that weren't used in any way.
It likely happened to be this way in a refactor and copy-pasted without much thinking.

### CODE REVIEW

**Review by shilman — APPROVED** [review]
Looks like these are also used in `builder-webpack5` -- can you please clean those up as well?

File: lib/builder-webpack4/package.json [hunk 1]
```diff
@@ -39,27 +39,6 @@
     "prepare": "node ../../scripts/prepare.js"
   },
   "dependencies": {
-    "@babel/core": "^7.12.10",
```
Comment (ndelangen) [inline lib/builder-webpack4/package.json:42]: This line needs to be reverted, because babel-loader has a peerDependency on it

**Comment by ndelangen** [comment]
good to merge, I think @shilman

**Comment by shilman** [comment]
@ndelangen examples CI is failing ❌

**Comment by ndelangen** [comment]
@shilman NOW it's good to merge, I think

**Pull request merged**